### ISSUE TITLE
Ability to retrieve Play Store last updated date?

### ISSUE THREAD

**Issue by SeBsZ** [NoMore201/googleplay-api#40]
Hey,

I'm wondering if it's possible to make an API call to retrieve the date an app was last updated on the Play Store? It's listed on the Play Store under additional information where it says "Updated": eg. https://play.google.com/store/apps/details?id=com.android.chrome

I looked through the protobuf definition as well as googleplay.py and googleplay_pb2.py but I doesn't look like this field is currently returned.

Does anyone have an idea? Thanks in advance!

**Comment by NoMore201** [comment]
DocV2 -> DocumentDetails -> AppDetails there's `uploadDate` field.

https://github.com/NoMore201/googleplay-api/blob/6f5565bdefb8cc9852240e04d9e3ea603bccfc73/googleplay.proto#L450-L473

In the `details()` and `bulkDetails()` functions I parse only a little part of this data. Maybe it is possible to expand the dictionary with this data if available

**Comment by matlink** [comment]
It is already available, I used it here : https://github.com/matlink/gplaycli/blob/master/gplaycli/gplaycli.py#L342

**Comment by archon810** [comment]
UploadDate is different from last updated date, since the latter can be triggered by the developer updating the some metadata, like app description or changelog, no?

**Comment by SeBsZ** [comment]
Yeah, I meant last updated date, not the uploaded date. Thanks though.
